packages/visx-tooltip/src/hooks/useTooltipInPortal.tsx

```diff
@@ -40,31 +43,44 @@ export default function useTooltipInPortal({
   const TooltipInPortal = useMemo(
     () =>
       ({
-        left: containerLeft = 0,
-        top: containerTop = 0,
+        left: tooltipLeft = 0,
+        top: tooltipTop = 0,
         detectBounds: detectBoundsProp, // allow override at component-level
+        portalContainer: portalContainerProp, // allow override at component-level
         zIndex: zIndexProp, // allow override at the component-level
         ...tooltipProps
       }: TooltipInPortalProps) => {
         const detectBounds = detectBoundsProp == null ? detectBoundsOption : detectBoundsProp;
+        const portalContainer =
+          portalContainerProp == null ? portalContainerOption : portalContainerProp;
+        const portalContainerRect = portalContainer?.getBoundingClientRect();
```

Comment:
this can be quite expensive to call each render which is why we opted to use `useMeasure` for the `container`. 

Is it necessary to support the `portalContainerProp` override or can it be passed in as an option? If the latter is fine we could use `useMeasure` for the `portalContainer` as well. that's tricky though because it may not always update when an element resize happens etc. (which is what `forceRefreshBounds` is for)

if this simplifies things we could leave it for now.

Reply:
Oh that's a good catch. 🤔 I don't think we necessarily need to support the override at the component level since typically a portal container doesn't move in the screen. I'll drop this prop at the tooltip level and just keep it on `useTooltipInPortal`.

However, I'll keep passing in the `HTMLDivElement` directly to `useTooltipInPortal` and memoize the bounding rect instead of using `useMeasure` for this if that's OK, because having to deal with the old fashioned ref `useMeasure` returns makes things harder on the consumer side in my opinion.